### PR TITLE
chore: avoid reporting errors for bad requests to profile endpoint

### PR DESCRIPTION
**Pull request by 1emu**

### CODE REVIEW

**Review by andyesp — APPROVED** [review]

File: src/hooks/useGovernanceProfile.ts [hunk 1]
```diff
@@ -8,6 +9,10 @@ import { ErrorCategory } from '../utils/errorCategories'
 
 import { DEFAULT_QUERY_STALE_TIME } from './constants'
 
+function isNotFoundOrBadRequest(error: RequestError) {
+  return error.statusCode === 404 || error.statusCode === 400
```
Comment (andyesp) [inline src/hooks/useGovernanceProfile.ts:13]: Why would this be a bad request?

Comment (1emu) [reply src/hooks/useGovernanceProfile.ts:13]: `validateAddress` throws BadRequest errors for invalid addresses (we don't need to report those)

File: src/hooks/useGovernanceProfile.ts [hunk 2]
```diff
@@ -19,7 +24,7 @@ export default function useGovernanceProfile(profileAddress?: string | null) {
         return await Governance.get().getUserProfile(profileAddress)
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
       } catch (error: any) {
-        if (error.statusCode !== 404) {
+        if (!isNotFoundOrBadRequest(error)) {
```
Comment (ncomerci) [inline src/hooks/useGovernanceProfile.ts:27]: are we sure that error is always of type `RequestError`?

**Review by ncomerci — APPROVED** [review]

File: src/hooks/useGovernanceProfile.ts (hunk 2 above)
Comment (1emu) [reply src/hooks/useGovernanceProfile.ts:27]: It should do so now, with the changes in the back

**Pull request merged**